### PR TITLE
Fix issues with Markdown Lint GitHub Action

### PR DESCRIPTION
**Pull request by jason-fox**
Add missing text-lint package. The latest text-lint complains if you add something like *comments* in the config, but don't include the module.

### CODE REVIEW

**Comment by jason-fox** [comment]
@fgalan  WTF Coveralls???? 😖  I didn't touch the code. How can code coverage go down ????

File: package.json [hunk 1]
```diff
@@ -77,6 +77,7 @@
     "should": "13.2.3",
     "sinon": "~9.0.2",
     "textlint": "~11.7.6",
+    "textlint-filter-rule-comments": "^1.2.2",
```
Comment (fgalan) [inline package.json:80]:
```suggestion
    "textlint-filter-rule-comments": "~1.2.2",
```

**Comment by fgalan** [comment]
> @fgalan WTF Coveralls???? 😖 I didn't touch the code. How can code coverage go down ????

I have observed that effect sometimes. A small oscillation around +/-0.05% in PRs. Maybe is some kind of race condition in tests (so some times more/less lines are traversed depending on the case. Ugly.

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**